src/components/FormValidator.js

```diff
@@ -78,7 +78,13 @@ export default class FormValidator {
   enableValidation() {
```

Comment:
You can add a `reset` listener here to   disable the submit button.  

Please, add this code here

```js

    this._formEl.addEventListener("reset", () => {
      this.disableButton();
    });


```

In this `reset` listener you are waiting for a special `reset` event which is fired when you `reset` any form in class `PopupWithForm` 

More info about `reset` is here: https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset_event